packages/interpretations/src/components/InterpretationsComponent/InterpretationsComponent.js

```diff
@@ -8,14 +8,8 @@ import pick from 'lodash/fp/pick';
 import { getFavoriteWithInterpretations } from '../../api/helpers';
 import Details from '../DetailsPanel/Details';
 import InterpretationsCard from '../Cards/InterpretationsCard';
-import i18n from '../../locales';
```

Comment:
Unfortunately I think this import needs to still be somewhere - it's a module with sideeffects (it adds the .json resources on load, should be added to the `package.json` [`sideEffects` array](https://webpack.js.org/guides/tree-shaking/).  I think if we remove this import the locale strings for this package will never get loaded?  Unless of course it's imported somewhere else (possible, I didn't search) in which case this is fine!

Reply:
See [the locales/index.js template](https://github.com/dhis2/app-platform/blob/master/cli/src/lib/i18n/templates/locales.hbs#L15-L18)